ci/azure/linux.yml

```diff
@@ -12,22 +12,20 @@ jobs:
     AZURECI: 1
     COMPOSE_FILE: ci/docker-compose.yml
     PYTHONHASHSEED: "random"
-    # TODO: omniscidb should be in BACKENDS_SQL_PARQUET, but it's not compatible with Python 3.8, so it needs to be set individually in py36 and py37 builds.
-    # It is not being added, because the conda solver takes forever to resolve when pymapd is present, so it's not being tested for now
-    BACKENDS_SQL_PARQUET: "mysql postgres sqlite parquet"
+    BACKENDS_SQL_PARQUET_OMNISCI: "mysql postgres sqlite parquet omnisci"
     BACKENDS_IMPALA_KUDU_CLICKHOUSE: "impala kudu-master kudu-tserver clickhouse"
     # TODO: there are some issues with pyspark, so not testing for now. Same for udf I guess, which was already not being tested.
-    PYTEST_MARK_EXPRESSION_SQL_PARQUET: "not udf and not spark and not pyspark and not clickhouse and not impala and not kudu and not hdfs"
-    PYTEST_MARK_EXPRESSION_IMPALA_KUDU_CLICKHOUSE: "not udf and not spark and not pyspark and not mysql and not parquet and not postgresql and not postgis and not postgres_extensions and not sqlite"
+    PYTEST_MARK_EXPRESSION_SQL_PARQUET_OMNISCI: "not udf and not spark and not pyspark and not clickhouse and not impala and not kudu and not hdfs"
+    PYTEST_MARK_EXPRESSION_IMPALA_KUDU_CLICKHOUSE: "not udf and not spark and not pyspark and not mysql and not parquet and not postgresql and not postgis and not postgres_extensions and not sqlite and not omniscidb"
   strategy:
     matrix:
       py36_backends1:
         PYTHON_MAJOR_VERSION: "3"
         PYTHON_MINOR_VERSION: "6"
         PYTHON_VERSION: $(PYTHON_MAJOR_VERSION).$(PYTHON_MINOR_VERSION)
         PYTHON_NO_DOT_VERSION: $(PYTHON_MAJOR_VERSION)$(PYTHON_MINOR_VERSION)
-        PYTEST_MARK_EXPRESSION: $(PYTEST_MARK_EXPRESSION_SQL_PARQUET)
-        BACKENDS: $(BACKENDS_SQL_PARQUET)
+        PYTEST_MARK_EXPRESSION: $(PYTEST_MARK_EXPRESSION_SQL_PARQUET_OMNISCI)
```

Comment:
can you break to *another* backend? the reason is that then sqlserver can use it (and we are not overloaded on anyone)

Reply:
I created another build group, I think that's what you meant. Not sure I understand about sqlserver, but having a group for omnisci and spark is convenient, because then we can simply not have that in the matrix for Python 3.8, and things are less hacky.

Let me know if this is not what you meant.

Reply:
ye that’s what i meant